### DISCUSSION TITLE
ALS skeleton retarget pose for Manny

### DISCUSSION THREAD

**Discussion by woodencase01** [Sixze/ALS-Refactored#357] [Q&A]
The fingers were rotated in a strange way. I started fixing them, but they were all similar to the pinky.

Would you like me to export the retargeter and the poses so it can be updated?

![image](https://github.com/Sixze/ALS-Refactored/assets/4787041/57501797-0c6c-4970-877f-7ea0651d9d84)

**Comment by dungeon2567** [comment]
Has anyone fixed it?
